libflux/src/core/semantic/builtins.rs

```diff
@@ -191,7 +191,7 @@ pub fn builtins() -> Builtins<'static> {
                 "#,
             },
             "internal/gen" => semantic_map! {
-                "tables" => "forall [t0] (n: int, tags: [{name: string | cardinality: int}]) -> [{_time: time | _value: float | t0}]",
+                "tables" => "forall [t0] (n: int, ?nulls: float, tags: [{name: string | cardinality: int}]) -> [{_time: time | _value: float | t0}]",
```

Comment:
While you're here, I've been meaning to fix this. Can you make tags optional? It's supposed to be optional but the type system has it specified incorrectly.